src/ziggurat/mapper.clj

```diff
@@ -1,6 +1,5 @@
 (ns ziggurat.mapper
```

Comment:
There would be other places where sentry errors are being reported in Ziggurat as well apart from the mapper namespace like messaging, middleware etc.

Reply:
Done. Need to although write the testcases for them because the coverage has dropped down.

Reply:
Cool thanks